packages/api/src/external/commonwell/link/shared.ts

```diff
@@ -47,48 +46,16 @@ export function patientWithCWData(
   return patientWithCW;
 }
 
-async function shouldDowngradeLink(link: NetworkLink): Promise<boolean> {
+function isInsideOrgExcludeList(link: NetworkLink, orgIdExcludeList: Set<string>): boolean {
   const identifiers = link.patient?.identifier || [];
-  const checkPromises = identifiers.map(async id => {
+  return identifiers.some(id => {
     const idSystem = id.system?.replace(/^urn:oid:/, "");
-    if (idSystem) {
-      return (await getCQDirectoryEntryById(idSystem)) !== undefined;
+    if (idSystem && orgIdExcludeList.has(idSystem)) {
+      console.log(`OrgID ${idSystem} is in the exclude list.`);
```

Comment:
NABD, but good to add a TODO to remove this next time we touch this file after the initial tests